src/Salad.java

```diff
@@ -0,0 +1,73 @@
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Iterator;
+
+public class Salad {
+    private String name;
+    ArrayList<Vegetable> ingredients;
+    private int saladCalories = 0;
+
+    Salad(String name, Vegetable... ingredient) {
```

Comment:
if you cannot make a salad without ingredients, than you should avoid Arbitrary Number of Arguments